src/ds3_cli/commands/fileSizeReadCloser.go

```diff
@@ -3,35 +3,36 @@ package commands
 import "os"
 
 // Implements the SizedReadCloser interface for a file.
-type fileSizeReadCloser struct {
+// Decorates an os.File and adds a Size function which determines the length of the file
+type fileWithSizeDecorator struct {
```

Comment:
Should this file be renamed as well?

fileSizeReadDCloser.go -> fileWithSizeDecorator.go?